bin/satellite-clone

```diff
@@ -60,6 +66,9 @@ def main():
         ansible_args = []
         ansible_args.append("--list-tasks")
         confirm = True
+    if args.flush_cache:
+        ansible_args = []
```

Comment:
This will overwrite any previous args, above we do that for `--list-tasks` since that is an independent task and won't run the playbook. If the playbook still runs when running `--flush-cache`, we should keep any other options (i.e. --verbose). You can remove this line if my above assumptions are correct

Reply:
Didn't realize that, updated, thanks